internal/app/machined/internal/phase/rootfs/mount_overlays.go

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package rootfs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/talos-systems/talos/internal/app/machined/internal/phase"
+	"github.com/talos-systems/talos/internal/app/machined/internal/platform"
+	"github.com/talos-systems/talos/internal/app/machined/internal/runtime"
+	"github.com/talos-systems/talos/pkg/userdata"
+
+	"golang.org/x/sys/unix"
+)
+
+const (
+	// SystemVarPath is the path to write runtime system related files and
+	// directories.
+	SystemVarPath = "/var/system"
```

Comment:
Should we move this to internal/pkg/constants? Seems like this and roots.etc might be good candidates for having these bits defined there.

Reply:
I debated that and what I landed on is that I'm starting to favor the idea of constants that are relevant to a particular domain (rootfs) in this case should live in that package. That being said I think the the global `constants.go` file has its' place as well. So just a long winded way of saying "I'm torn". :)